### ISSUE TITLE
About this Project

### ISSUE THREAD

**Issue by tylike** [Apollo3zehn/BlazorMonacoRoslyn#1]
hi,

Will this project be updated?
I searched here when I studied Roslyn + Monaco.
I found that provideHover and provideSignatureHelp and other functions are half realized.

Besides, I'm https://github.com/dotnet/aspnetcore/issues/28689 I also saw the problems you encountered here, and I also reproduced them. Can I prevent jitter and throttling on the JS side?

https://docs.microsoft.com/en-us/aspnet/core/blazor/performance?view=aspnetcore -6.0#optimize JavaScript interop speed
"Don't trigger events too quickly."
It is mentioned in the chapter.

thanks!

**Comment by Apollo3zehn** [comment]
Hi, 
I am sorry but I am not actively working on this project. I created it only to share my progress trying to create a browser editor for another project (to let users create small plugins quickly). In the meantime I have factored out the need for an online editor and now I am trying to create a VSCode extension instead that connects to my software and supports the user to write these extensions.

The main projects implementation right before I removed support for Monaco is here https://github.com/Nexusforge/nexus/tree/c3461400a56eb3c7972824f5370643fd4e1bdafa/src/Nexus.Core , here https://github.com/Nexusforge/nexus/blob/c3461400a56eb3c7972824f5370643fd4e1bdafa/src/Nexus.Core/Extensions/DataSource/FilterDataSource.cs#L332 and here https://github.com/Nexusforge/nexus/blob/c3461400a56eb3c7972824f5370643fd4e1bdafa/src/Nexus/Pages/FilterEditor.razor#L46

I should archive the project, to make the current status more clear.
